frontend/src/components/widgets/card-grid-view/CardGridView.css

```
@@ -0,0 +1,1152 @@
+  border: 1px solid #d9d9d9;
+}
+
+/* Expanded Content */
+.card-expanded-content {
+  margin-top: 16px;
+  padding-top: 16px;
+  border-top: 1px solid #f0f0f0;
+  animation: cardExpandAnimation 0.2s ease-out;
+}
+
+@keyframes cardExpandAnimation {
```

Stylelint (17.3.0)

[error] Expected keyframe name "cardExpandAnimation" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)